sphinx_mdolab_theme/ext/embed_n2.py

```diff
@@ -84,7 +84,7 @@ def run(self):
         object_tag = (
             "<iframe width='" + str(n2_dims[0]) + "'"
             " height='" + str(n2_dims[1]) + "'"
-            " style='border: 1px solid lightgray; resize: both;'"
+            " style='border: 0px solid lightgray; resize: both;'"
```

Comment:
Does this fully fix the weird whitespace issue?

Reply:
No it doesn't fix the problem, but it does get rid of the grey border around the HTML that I found ugly

Reply:
Hmm okay, do you want to debug that more or just leave as is? My guess is that if you view the HTML source code on that page, you may be able to figure out what's going on.

Reply:
I'd like to leave it as is. I tried a bit with changing the style and other arguments but couldn't figure it out. It's possible that it has to do with OpenMDAO's N2 HTML code because it autoscales to the window and sometimes is buggy.